Mathlib/Order/WellFoundedSet.lean

```diff
@@ -694,6 +694,19 @@ theorem IsWF.min_union (hs : s.IsWF) (hsn : s.Nonempty) (ht : t.IsWF) (htn : t.N
 
 end LinearOrder
 
+section LocallyFiniteOrder
+
+variable {s : Set α} [Preorder α] [LocallyFiniteOrder α]
+
+theorem bddBelow_wellFoundedOn_lt : BddBelow s → s.WellFoundedOn (· < ·) := by
```

Comment:
Please move it out of the `Set` namespace and call it
```suggestion
theorem BddBelow.wellFoundedOn_lt : BddBelow s → s.WellFoundedOn (· < ·) := by
```
This will allow dot notation.

Also, please add the dual lemma `BddAbove.wellFoundedOn_gt` (you can prove it by `h.dual.wellFoundedOn_lt`).

Reply:
It was `BddBelow.`, not `BddBelow_`. Otherwise, dot notation won't work.

Reply:
Oops, sorry about that oversight.  I was wondering why `h.dual.wellFoundedOn_lt` didn't work.